qa/automation/CareerPortal.Tests.Scenarios/4. Sending CV/4.1.Sending CV.feature

```diff
@@ -0,0 +1,132 @@
+Feature: 4.1. Sending CV
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 4.1_01 Correct display of the cv submit form
+    Given user wanted to send CV
+    When user opens "<name of the page>" the page
+    Then user finds the form for sending CV
+	And the form includes "Name", "Email", "Message" fields
+	And the form includes "Attach CV" and "Send" buttons
+	And the form includes checkbox "I agree to the personal data processing"
+
+    Examples: 
+		| name of the page                                               |
+		| https://career.quantori.com/ru                                 |
+		| https://career.quantori.com/ru/positions/cloud-python-engineer |
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 4.1_02 Successful submission with filled "Name", "Email" fields, checkbox and attached CV
+    Given user founded the Send CV form on page "<name of the page>"
+    And user entered "<Name>", "<Email>", "<Attach CV>", "<Checkbox>" fields with foolowing combination of values:
+        | Name               | Email               | Attach CV   | Checkbox |
+        | Evgeniia T         | tikeeva_es@mail.ru  | cv.pdf 10Mb | True     |
+        | Евгения Т          | tikeeva1998@mail.ru | cv.doc 2Mb  | True     |
+        | Evgeniia T S       | tikeeva_es@mail.ru  | cv.docx 2Mb | True     |
+        | Ж'еня-Тикеева, мл. | tikeeva1998@mail.ru | cv.rtf 2Mb  | True     |
+	When user clicks "Send" button
+    Then user should see the message that CV was submitted
+
+    Examples: 
+		| name of the page                                               |
+		| https://career.quantori.com/ru                                 |
+		| https://career.quantori.com/ru/positions/cloud-python-engineer |
+
+@Type:Positive
```

Comment:
Scenario 4.1_02 dublicatates of 4.1_03. I would be choose 4.1_03  but with following edits :
In 39 row after word "values" I would be delete ":"
You missed key word "Examples" before you table of values ( see attach)
Also for max coverage I would be add cv file with size 9,9 Mb to test that our form will be send
![111](https://user-images.githubusercontent.com/94995338/144238383-c357b049-e220-4960-89e3-c4df352f0312.JPG)